docker-compose.yml

```diff
@@ -10,37 +10,44 @@ services:
       KEYCLOAK_USER: admin
       KEYCLOAK_PASSWORD: admin
       DB_VENDOR: h2
-  postgres:
+  metricsdb:
     image: postgres:9.6
-    container_name: postgres
-    volumes:
-    - "./config/postgres.sql:/docker-entrypoint-initdb.d/postgres.sql"
-    ports:
-      - "5432:5432"
+    container_name: metricsdb
+    environment:
+      POSTGRES_USER: metrics	
+      POSTGRES_PASSWORD: metrics	
```

Comment:
not sure if it's my `docker-compose` version, but these tabs make the command to fail:
```
docker-compose up -d
ERROR: yaml.scanner.ScannerError: while scanning for the next token
found character '\t' that cannot start any token
  in "./docker-compose.yml", line 17, column 29
```
lines: 17, 18 & 36